static/js/pivotal.js

```diff
@@ -143,21 +154,80 @@
         'X-TrackerToken': pt_token
       },
       success: function(data) {
-        // Find all git commit messages
-        var list = data.filter(function(obj) {
-          return obj.commit_type;
-        });
-        // Find repo name from comment body text
-        list = list.map(function(item) {
-          return item.text.split('https://github.com/gengo/')[1].split('/commit/')[0];
-        });
+        var PULL_REQEST_REGEX = /Merge pull request/;
+        var COMMIT_REPO_REGEX = /https:\/\/github.com\/gengo\/(.+)\/commit\//;
+        var DEPLOY_REPO_REGEX = new RegExp('Deployed (.+) to '+ config.environment +': ');
+
+        var list = data.filter(function(activity) {
```

Comment:
`list` doesn't really tell us what this variable is about, except its type. Maybe `activities` would be a clearer name ?